internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -443,6 +429,54 @@ func (mc *mac) triggerDefaultStream(w http.ResponseWriter, r *http.Request, line
 	return true
 }
 
+func (mc *mac) generateRouteKeys(stream *livepeer.CreateStreamResp) []string {
+	serviceName := mc.routePrefix + serviceNameFromMistURL(mc.mistURL)
+	wildcardPlaybackID := mc.wildcardPlaybackID(stream)
+	playbackID := mc.routePrefix + stream.PlaybackID
+
+	keys := []string{
+		traefikKeyPathServices + serviceName + "/loadbalancer/servers/0/url",
+		mc.mistURL,
+		traefikKeyPathServices + serviceName + "/loadbalancer/passhostheader",
+		"false",
+	}
+
+	// Add a millisecond timestamp as the priority so new streams always win over old, stale streams
+	priority := strconv.FormatInt(time.Now().UnixNano()/int64(time.Millisecond), 10)
+
+	for _, prefix := range []string{"hls", "cmaf"} {
```

Comment:
**nit:** Maybe have a "const" for this slice like `var playbackProtoPrefixes = []string{"hls", "cmaf"}` or something like that? (later could even become cli flags as well? 🤔)

Reply:
Done - originally tried to do it with a `const` and it didn't work, didn't occur to me to just make a package-scoped `var` 🤦‍♂️